README.md

```diff
@@ -6,7 +6,7 @@
   <h3 align="center">Anyxz</h3>
 
   <p align="center">
-    Realtime Video Background Semantic Segmentation in Browser ⚡
+    <b>Realtime Video Background Semantic Segmentation in Browser ⚡ </b>
```

Comment:
Don't make it bold. The title should always be in bold. You can tweak with the font-size of the title.